nav_output.txt

```diff
@@ -0,0 +1,1006 @@
+stopping
```

Comment:
can this be added to a .gitignore?

Reply:
I'll just delete these files, they're not important